### ISSUE TITLE
anti-chromatin antibody

### ISSUE THREAD

**Issue by anastasia-coding** [obophenotype/human-phenotype-ontology#11343]
**Preferred term label:**
anti-chromatin antibodies

**Synonyms**
anti‐nucleosome antibodies

**Definition (free text, please give PubMed ID)**
Anti-chromatin antibodies are autoantibodies directed against nucleosomes (complexes of DNA and histone protein) and are associated with systemic lupus erythematosus (PMID: 17763207)

**Parent term (use hpo.jax.org/app)**
Antinuclear antibody positivity

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Systemic Lupus Erythematosus (OMIM:152700)
 
**Your nano-attribution (ORCID)**
Anastasia-Vasiliki Madenidou (https://orcid.org/0000-0002-7382-9958)

**Comment by pnrobinson** [comment]
This is a synonym for [Anti-H2A-H2B antibody positivity HP:0033593](https://hpo.jax.org/browse/term/HP:0033593).
Todo: Update definition and synonym list according to https://pubmed.ncbi.nlm.nih.gov/12848987/

**Comment by MaudDLVT** [comment]
Dear @pnrobinson ,
Thank you for your email. I'm replying directly here, I hope that's OK.
Anti-chromatin antibody positivity is a class above anti-H2A and anti-H2B antibody positivity. It is less specific. 
Potentially, other autoantibodies can also be added to this very large category of anti-chromatin antibodies.
"anti‐nucleosome antibodies positivity" is not an equivalent term and should not be considered as a synonym.
Nucleosomes are part of chromatin, but chromatin has other components as well.
Best, Maud

**Comment by pnrobinson** [comment]
@MaudDLVT @anastasia-coding 

Adding as
Antichromatin antibody positivity
Def:
The presence of autoantibodies (immunoglobulins) in the blood circulation that react against the chromatin. Human autoantibodies that bind to chromatin targets can be divided into those that recognize dsDNA, protein components of chromatin (i.e., histones, HMG proteins), mononucleosomes, or macromolecular components of nucleosomes as represented by low salt extracted nucleosomes (core particle).
Comment:
Eukaryotic chromatin is comprised of approximately 40% DNA, 40% histones, 20% nonhistone proteins (i.e., HMG proteins), RNA, and other macromolecules.
Citation: https://pubmed.ncbi.nlm.nih.gov/24804269/
